src/Concordium/Client/Runner.hs

```diff
@@ -2500,6 +2365,683 @@ generateBakerKeys bkBakerId = do
+        let decreaseOrRemove = if removing then "Removing a baker" else "Decreasing the amount a baker is staking"
+        logWarn [decreaseOrRemove ++ " will lock the stake of the baker for a cooldown period before the CCD are made available."]
+        logWarn ["During this period it is not possible to update the baker's stake, or stop the baker."]
+        logWarn [[i|The current baker cooldown would last until approximately #{cooldownDate}|]]
+        let confirmStr = if removing then "remove the baker" else "update the baker's stake"
+        confirmed <- askConfirmation $ Just $ "Confirm that you want to " ++ confirmStr
+        unless confirmed exitTransactionCancelled
+
+    warnIfCapitalIsBig capital amount =
+      when ((capital * 100) > (amount * 95)) $ do
+        logWarn ["You are attempting to stake >95% of your total CCD on this account. Staked CCD is not available for spending."]
+        logWarn ["Be aware that updating or stopping your baker in the future will require some amount of non-staked CCD to pay for the transactions to do so."]
+        confirmed <- askConfirmation $ Just "Confirm that you wish to stake this much CCD"
+        unless confirmed exitTransactionCancelled
+
+    transactionForBakerConfigure confirm = do
+      baseCfg <- getBaseConfig baseCfgDir verbose
+      when verbose $ do
+        runPrinter $ printBaseConfig baseCfg
+        putStrLn ""
+      let cbMetadataURL = fmap (Types.UrlText . Text.pack) metadataURL
+      (bakerKeys, cbKeysWithProofs) <- readInputKeysFile baseCfg
+      let payload = Types.encodePayload Types.ConfigureBaker{..}
+          nrgCost _ = case cbKeysWithProofs of
+                        Nothing -> return . Just $ bakerConfigureEnergyCostWithoutKeys (Types.payloadSize payload)
+                        Just _ -> return . Just $ bakerConfigureEnergyCostWithKeys (Types.payloadSize payload)
+      txCfg@TransactionConfig{..} <- getTransactionCfg baseCfg txOpts nrgCost
+      logSuccess
+        ([ printf "configuring baker with account %s" (show (naAddr $ esdAddress tcEncryptedSigningData))
+         , printf "allowing up to %s to be spent as transaction fee" (showNrg tcEnergy) ]
+         ++ configureCapitalLogMsg
+         ++ configureRestakeLogMsg
+         ++ configureOpenForDelegationLogMsg
+         ++ configureTransactionFeeCommissionLogMsg
+         ++ configureBakingRewardCommissionLogMsg
+         ++ configureFinalizationRewardCommissionLogMsg)
+      when confirm $ do
+        confirmed <- askConfirmation Nothing
+        unless confirmed exitTransactionCancelled
+      when verbose $ do
+        runPrinter $ printSelectedKeyConfig $ tcEncryptedSigningData
+        putStrLn ""
+      return (bakerKeys, txCfg, payload)
+
+    configureCapitalLogMsg =
+      case cbCapital of
+        Nothing -> []
+        Just capital -> [printf "stake will be %s CCD" (Types.amountToString capital)]
```

Comment:
```suggestion
        Just capital -> [[i|stake will be #{Types.amountToString capital} CCD|]]
```
When adding new code that uses string interpolation, please use this form (https://hackage.haskell.org/package/string-interpolate). `printf` is not safe.
You don't need to change all the occurrences now. Just use the string-interpolate package in the future ;)